File: Bookworm Backend Dot net/Models/Language.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;
using System;

namespace Bookworm.Models
{
    [Table("language_master")]
    public class Language
    {
        [Key]
        [Column("language_id")]
        public int Id { get; set; }

        [StringLength(50)]
        [Column("language_desc")]
        public string? Description { get; set; }
    }
}

File: Bookworm Backend Spring Boot/src/main/java/com/bookworm/entities/Language.java
package com.bookworm.entities;

import java.time.LocalDate;
import java.util.Set;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

/**
 * Represents a language for products.
 * Maps to the 'languages' table.
 */
@Entity
@Table(name = "language_master")
@Getter
@Setter
public class Language {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "language_id")
    private Integer id;

    @Column(name = "language_desc", length = 50) // Increased from 5 in ERD for practicality
    private String description;

}